src/contrib/debugging-sssd.rst

```diff
@@ -206,3 +211,80 @@ i.e. ``su admin@ipa.test``. Then start gdb and attach it to the server:
     debugging the child once it is forked off the parent. See `gdb manual
     <https://sourceware.org/gdb/onlinedocs/gdb/Forks.html>`__ for more
     information.
+
+Debugging KCM
+*************
+KCM is different from other SSSD processes in the way it is launched. It is not
+forked by the monitor as all the other processes, but it is launched by a
+``systemd trigger unit`` when a client application (like ``klist``) opens its
+socket. After 5 minutes of inactivity it will automatically shut down.
```

Comment:
if `tgt_renewal = true` then we disable the responder idle timeout, maybe not worth mentioning though